src/main/java/Command.java

```diff
@@ -0,0 +1,170 @@
+import java.io.IOException;
+
+class Command {
+
+    /**
+     * constructor.
+     * @param tasks is the list of tasks.
+     * @param ui is the UI class.
+     * @param storage is the Storage class.
+     * @throws DukeException when input is different from required.
+     * @throws IOException when input is different from required.
+     */
+    void execute(TaskList tasks, Ui ui, Storage storage) throws DukeException, IOException{
```

Comment:
Rather than putting a method with empty body, isn't it better to declare Command as an abstract class?